API.md

```diff
@@ -43,6 +51,8 @@ in the [API.md](https://github.com/containers/libpod/blob/master/API.md) file in
 
 [func HistoryImage(name: string) ImageHistory](#HistoryImage)
 
+[func ImageExists(name: string) int](#ImageExists)
```

Comment:
Why is this better than GetImage() and checking for an error?

Reply:
Probably from an API point of view, it is not.  The exists calls were added to differentiate between errors of no exist versus ,  you can not check if container exists.

So i the GetImage returns an error code that can be determined to be Container does not exist versus permission denied, no need for the new API.